src/mixer/playermanager.h

```diff
@@ -188,9 +188,8 @@ class PlayerManager : public QObject, public PlayerManagerInterface {
     // Slots for loading tracks into a Player, which is either a Sampler or a Deck
     void slotLoadTrackToPlayer(TrackPointer pTrack, const QString& group, bool play = false);
     void slotLoadLocationToPlayer(const QString& location, const QString& group, bool play = false);
-    void slotLoadLocationToPlayerStopped(const QString& location, const QString& group) {
```

Comment:
The default play in the two functions above should be probably removed to make it explicit.